templates/zerver/create_realm.html

```diff
@@ -36,6 +55,11 @@ <h1 class="get-started">{{ _("Create a new Zulip organization") }}</h1>
                 <div class="alert alert-error">{{ error }}</div>
                 {% endfor %}
             {% endif %}
+            {% if form.errors.__all__ %}
```

Comment:
This duplicates the loop above -- try submitting without an email address, and you see two copies of the error.

Reply:
Hmm. It shouldn't. `__all__` catches only those errors that are not associated with any field. Let me add a test for this as well.

Ref: https://docs.djangoproject.com/en/2.0/ref/forms/validation/#validating-fields-with-clean

Reply:
However, I do get duplicate errors when I try to re-submit. Interesting!

Reply:
The problem was client side validation. Fixed.